packages/react/src/components/Workbook/index.tsx

```diff
@@ -291,6 +297,158 @@ const Workbook = React.forwardRef<WorkbookInstance, Settings & AdditionalProps>(
       [emitOp]
     );
 
+    // 更改页面缩放
+    const changeZoom = useCallback(
+      (operate: string) => {
```

Comment:
It's not a good practice to use string as "operation" switch. In this case, pass in a delta is better
`const zoomBy = useCallback((delta: number) => {}`

and call `zoomBy(0.1)` and `zoomBy(-0.1)`

Also, try to use `!` as less as possible.